### ISSUE TITLE
Plugin crashes when using with cordova-email-plugin

### ISSUE THREAD

**Issue by blablapolicja** [katzer/cordova-plugin-app-event#3]
plugin crashes when I call cordova.plugins.email.isAvailable() or cordova.plugins.email.open()

<img width="861" alt="2016-01-07 14 53 45" src="https://cloud.githubusercontent.com/assets/7767522/12172753/198ffbfc-b553-11e5-9e25-33c8f60223b1.png">

my code:

``` javascript
$scope.goToStatistics = function (){
    if (window.cordova) {
        cordova.plugins.email.isAvailable(function (isAvailable) {
            console.log(isAvailable);
        });
    }
};
```

I tried to wrap it in $ionicPlatform.ready(), but nothing changed
Everything is ok, when I put plugin method in app.run()

``` javascript
.run(function ($ionicPlatform) {
    $ionicPlatform.ready(function() {
        cordova.plugins.email.isAvailable(function (isAvailable) {
            console.log(isAvailable);
        });
    });
})
```

Ionic 1.2.1
Cordova 5.4.1
cordova-ios 4.0.1
cordova-plugin-app-event 1.1.0
cordova-plugin-email 1.1.0

**Comment by zfoltin** [comment]
This plugin also causes issues when used with https://github.com/whiteoctober/cordova-plugin-app-version
pluginInitialize is not implemented there and since the plugin name does not start with 'CDV' https://github.com/katzer/cordova-plugin-app-event/blob/master/src/ios/CDVPlugin%2BAPPAppEvent.m#L42 it insists on calling orig_pluginInitialize https://github.com/katzer/cordova-plugin-app-event/blob/master/src/ios/CDVPlugin%2BAPPAppEvent.m#L57 regardless if it's implemented or not.

**Comment by zfoltin** [comment]
Probably duplicate of https://github.com/katzer/cordova-plugin-app-event/issues/1

**Comment by katzer** [comment]
Fixed with v1.2.0
